src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java

```diff
@@ -85,4 +85,40 @@ public User findByLogin(User user) {
         System.out.println("user is not exist");
         return userFind;
     }
+
+    @Override
+    public User getUserByLogin(String login) {
```

Comment:
We have two methods with the same logic, but different arguments. Can we reuse code and wrap one into other?